index.html

```diff
@@ -8,20 +8,23 @@
   </head>
   <body class="purple-1">
     <section class="menu purple-4">
-      <nav>
-        <img class="hamburger" src="assets/menu.svg" alt="menu">
+      <nav class="navigation">
+        <img class="menu-open" src="assets/menu.svg" alt="menu">
+        <img class="menu-close" src="assets/menu-close.svg" alt="menu-close"/>
```

Comment:
Nice work on adding classes to these HTML elements so that we can better target the styles.  I would change the class name of either .menu-open (12) or .open-menu (15) because they could be used interchangeably by accident.  Maybe consider changing .menu-open on line 12 to .hamburger, or changing .open-menu on line 15 to .open-nav.